adsb_docker_install.sh

```diff
@@ -1473,13 +1473,13 @@ function create_docker_compose_yml_file() {
             echo "    depends_on:"
             echo "      - readsb"
             echo "    environment:"
-            echo '      - ALT=${FEEDER_ALT_M}m'
+            echo "      - ALT=${FEEDER_ALT_M}m"
```

Comment:
These quotes need to be single quotes, as the intention here for the `docker-compose.yml` file to use variables from the `.env` file, not variables from the script.

Reply:
Ah! Okay, still learning. Just cancel it, I was just eager to help, but it is okay to cancel :-) Just change 932 and 941 then.